lib/src/values/logic_value.dart

```diff
@@ -18,6 +18,7 @@ typedef LogicValues = LogicValue;
 ///
 /// Each bit of [LogicValue] can be represented as a [LogicValue]
 /// of `0`, `1`, `x` (contention), or `z` (floating).
+/// LogicValue is unsigned
```

Comment:
Would be better to make this a new line and a sentence with a period, and put [] around it. 

 i.e.

```dart
/// An immutable 4-value representation of an arbitrary number of bits.
///
/// Each bit of [LogicValue] can be represented as a [LogicValue]
/// of `0`, `1`, `x` (contention), or `z` (floating).
///
/// [LogicValue] is unsigned.
```